test/demo/values.yaml

```diff
@@ -3,6 +3,16 @@
 # Includes components used in the demo, defaults to alpha3 rules.
 # Note: please only put common configuration for the demo profiles here.
 global:
+  istioNamespace: istio-system
```

Comment:
It would be really great if the demo used the split-namespace model. 
It's a demo, intended for fresh install/remove in non-prod clusters (since it takes over istio-system ) - no upgrade concerns.

Reply:
Is this still the plan with the new same-namespace upgrade plan you made? Or would that be each component upgrades in the same namespace, but we still have split namespaces?

Reply:
Let's go with the simplest - all in istio-system.